src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from 'styled-components'
+import { theme } from '../../theme'
+import { Label } from '../Label'
+
+const CheckboxContainer = styled.div`
+  display: inline-block;
+  vertical-align: middle;
+`
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  clip: rect(0 0 0 0);
+  clippath: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`
+
+const StyledCheckbox = styled.div<{ checked?: boolean }>`
+  display: inline-block;
+  border: 1px solid ${props => props.checked ? theme.primary : theme.grey50};;
+  margin-right: 10px;
+  width: 16px;
+  height: 16px;
+  background: ${props => props.checked ? theme.primary : 'transparent'};
+  display: flex;
+  transition: all 150ms;
+  ${HiddenCheckbox}:focus + & {
+    box-shadow: 0 0 0 3px ${theme.grey40};
+  }
+`
+
+const Icon = styled.svg`
+  fill: none;
+  stroke: ${theme.fullWhite};
+  stroke-width: 2px;
+`
+
+const HorizontalLabel = styled(Label)`
+  display: flex;
+  margin-bottom: 0px;
+  align-items: center;
+`
+
+export type CheckboxProps = {
```

Comment:
shouldn't we support `name`?  otherwise how will the value be submitted if the checkbox is used in a form?
what about other useful `input` props like `disabled`?